lib/miq_expression.rb

```diff
@@ -1139,7 +1139,10 @@ def self.model_details(model, opts = {:typ => "all", :include_model => true, :in
 
     result = []
     unless opts[:typ] == "count" || opts[:typ] == "find"
-      result = get_column_details(relats[:columns], model, model, opts).sort! { |a, b| a.to_s <=> b.to_s }
+      @column_cache ||= {}
+      key = "#{model}_#{opts[:interval]}_#{opts[:include_model] || false}"
```

Comment:
From looking at the code, those are the params that `get_column_details` is using.
Im also good with dropping the caching all together